src/test/deserialization_tests.cpp

```diff
@@ -193,6 +260,15 @@ TEST_P(DeserializeTFTensorProtoNegative, ShouldReturnNullptrForPrecision) {
                                << " should return nullptr";
 }
 
+TEST_P(DeserializeCAPITensorProtoNegative, ShouldReturnNullptrForPrecision) {
```

Comment:
The test is NEgative but named to be positive: `ShouldReturnNullptrForPrecision`